### PR TITLE
Add a8c reader icon

### PR DESCRIPTION
**Pull request by koke**
Adds missing icon for the Automattic section on the reader menu

simulator screen shot 27 sep 2016 10 19 17

Needs review: @aerych

### CODE REVIEW

**Comment by kwonye** [comment]
I know this only affects ~500 people, but thank you so much for doing this 👍

**Comment by aerych** [comment]
Chatted about this one a bit. 
@koke ping me when you've pushed the updates and I'll review :)

**Comment by koke** [comment]
@aerych ready for another review

File: WordPress/Classes/Models/ReaderTeamTopic.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+@objc public class ReaderTeamTopic : ReaderAbstractTopic
+{
+    @NSManaged public var slug: String
+
+    override public class var TopicType: String {
+        return "team"
+    }
+
+    public var icon: UIImage? {
+        guard bundledTeamIcons.contains(slug) else {
```
Comment (aerych) [inline WordPress/Classes/Models/ReaderTeamTopic.swift]: Nice touch

File: WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift [hunk 2]
```diff
@@ -77,8 +77,7 @@ import WordPressComAnalytics
     /// - Returns: True if the topic is for Freshly Pressed
     ///
     public class func topicIsFreshlyPressed(topic: ReaderAbstractTopic) -> Bool {
-        let path = topic.path as NSString!
-        return path.hasSuffix("/freshly-pressed")
+        return topic.path.hasSuffix("/freshly-pressed")
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift]: +1 for cleanup :)

**Comment by aerych** [comment]
Glorious. Tested and it works great. And tyvm for switching things up so the icon is referenced via slug. It lays the ground work for when teams is more widely implemented.
:shipit:

**Pull request merged**